lib/features/local_live_log.js

```diff
@@ -35,8 +43,13 @@ export default class TaskclusterLogs {
     let taskId = task.status.taskId;
     let runId = task.runId;
 
+    this.artifactPath = TaskclusterLogs.getArtifactPathFromTask(task.task);
+
+    let directory = path.parse(this.artifactPath).dir;
+    let backingPath = directory + '/' + BACKING_ARTIFACT_NAME;
```

Comment:
This is probably ok, but would look weird if someone named the live log something other than live.log.

For instance if task.payload.log is 'b2g-team/logs/task_log.log' the backing log will be 'b2g-team/logs/live_backing.log'

Sometimes makes me wonder if we either should:
A) parse file name, and add _backing to it
B) make take.payload.log be just a path to put logs under, such as setting it to 'b2g-team/logs'

I'm fine with this for now though as really the only real usecase is for the b2g specific stuff and the tasks can understand that this is what will happen.